cnd/src/http_api/connector.rs

```diff
@@ -0,0 +1,163 @@
+use crate::{
+    dependencies::Dependencies,
+    network::{DialInformation, Network, RequestError, SendRequest},
+    seed::{Seed, SwapSeed},
+    swap_protocols::{
+        asset::Asset,
+        metadata_store::{self, MetadataStore},
+        rfc003::{
+            self,
+            alice::SpawnAlice,
+            bob::SpawnBob,
+            create_ledger_events::CreateLedgerEvents,
+            state_machine::SwapStates,
+            state_store::{self, StateStore},
+            ActorState, Ledger,
+        },
+        LedgerConnectors, Metadata, SwapId,
+    },
+};
+use futures::{
+    sync::{mpsc, oneshot::Sender},
+    Future,
+};
+use libp2p::PeerId;
+use libp2p_comit::frame::Response;
+use std::sync::Arc;
+
+/// Collect all the connector trait bounds together under one trait.
+pub trait Connect:
+    Clone + MetadataStore + StateStore + Network + SendRequest + SpawnAlice + SpawnBob + SwapSeed
+{
+}
+
+#[allow(missing_debug_implementations)]
+pub struct Connector<S> {
```

Comment:
What is the difference between this and `http_api::Dependencies`?

Reply:
My bad, that file was supposed to be deleted.  This was re-named to `http_api::Dependencies`.